tests/vmidefaults_test.go

```diff
@@ -67,10 +69,11 @@ var _ = Describe("VMIDefaults", func() {
 
 		It("Should be applied to VMIs", func() {
 			// create the VMI first
-			err = virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(tests.NamespaceTestDefault).Body(vmi).Do().Error()
+			_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
```

Comment:
you should actually be able to see the changes right here in the returned VMI.

```newVmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)```

that's all that is needed now. You can get rid of the GET below.